lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -709,6 +709,9 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, layerID string, imageStore string) *containers.CreateParams {
 	config := &models.ContainerCreateConfig{}
 
+	config.NumCpus = &cc.HostConfig.CPUCount
+	config.Memory = &cc.HostConfig.Memory
```

Comment:
@jzt It was only standardized in 1998 and there's a lot of legacy usage (plus ingrained habits). MB used to mean 1024 but now means 1000 and MiB means 1024 (https://en.wikipedia.org/wiki/Mebibyte).

It appears that vSphere is using the original interpretation of MB/GB.